prime-router/src/main/java/fhirengine/translation/hl7/structures/nistelr251/datatype/CE.java

```diff
@@ -0,0 +1,208 @@
+/*
```

Comment:
nit: these are in the translation folder but have nothing to do with translation proper?

Reply:
Not sure I follow, this is the same package with the `HL7ToFhirTranslator`?